tests/language-feature/no-hlsl-pack-constant-buffer-elements.slang

```diff
@@ -0,0 +1,17 @@
+//TEST:SIMPLE(filecheck=CHECK):-target hlsl -entry main -profile cs_6_0 -no-hlsl-pack-constant-buffer-elements
+
```

Comment:
It would be better to add another test line here ensure the generated HLSL is indeed valid and can be accepted by DXC to produce dxil:
```
//TEST:SIMPLE: -target dxil -entry main -profile cs_6_0 -no-hlsl-pack-constant-buffer-elements
```

Reply:
I had to rework the test a tiny bit to make this work. Let me know if the changes look fine to you. Both generated test cases pass locally. 81b29d4f6bfaf97d5a691846cd49f15f6557d6af